irqbalance.h

```diff
@@ -101,6 +101,9 @@ extern int get_cpu_count(void);
 extern void rebuild_irq_db(void);
 extern void free_irq_db(void);
 extern void add_cl_banned_irq(int irq);
+#ifdef AARCH64
+extern void add_banned_irq(int irq);
+#endif
```

Comment:
Same here, with the bifurcation you've done, you can just export this without restricting it to only AARCH64

Reply:
fixed